.gitignore

```diff
@@ -15,8 +15,10 @@ test/e2e/data/infrastructure-docker/v1alpha3/cluster-template*.yaml
 test/e2e/data/infrastructure-docker/v1alpha4/cluster-template*.yaml
 test/e2e/data/infrastructure-docker/v1beta1/cluster-template*.yaml
 
-# E2e test extension deployment
+# E2e test extension deployment and patch files
 test/e2e/data/test-extension/deployment.yaml
+test/extension/config/default/extension_image_patch.yaml-e
+test/extension/config/default/extension_pull_policy.yaml-e
 
```

Comment:
Just to clarify those files are not generated. They are some side-effect of afaik an editor (or maybe something else?).

I also have them from time to time, but only very rarely (and up until now I only had them for the files which are overwritten when running `make docker-build`, e.g. `config/default/manager_image_patch.yaml`)

Or?

Reply:
Given the way we change those files I would guess some tmp files of sed

Reply:
I was trying to track down what created them - I assume it's either kustomize or intellij, but I still haven't figured it out 🙁.

Good to have them in the .gitignore anyway IMO - also have them for the manager*.yaml-e already.

Reply:
Good find! This seems to be an OSX issue, and I think it's well enough taken care of by adding it to gitignore. Might be a good idea for me to add gnu sed to my env though!

Reply:
I'm fine with just adding it, let's just do it for all (as you suggested) and let's add a short-comment where those files are ~ coming from

Reply:
How does the latest version look?

Reply:
Confirmed btw that gnu-sed on MacOS doesn't have this issue. Our flags for sed aren't fully portable. I'll create an issue for that to track it, but it's low impact and priority imo

Reply:
> How does the latest version look?

Perfect. I didn't know we already had the gitignore for manager YAMLs. But explains why you're only seeing that issue now